Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -352,9 +352,11 @@ namespace AzToolsFramework
         const AssetBrowserEntry* AssetBrowserTreeView::GetEntryByPath(QStringView path)
         {
             QModelIndex current;
-            for (auto token : AZStd::ranges::split_view(path, '/'))
+            bool currentlyFindingRows = false;
+            AZ::IO::Path p = path.toUtf8().constData();
```

Comment:
I think single letter variables names are against our coding standard. Could you change this to something like "pathString"?

Reply:
Yup will do, thanks!